tests/Functional/EventListener/FlashMessageListenerTest.php

```diff
@@ -21,10 +23,12 @@ class FlashMessageListenerTest extends WebTestCase
     public function testFlashMessageCookieIsSet()
     {
         $client = static::createClient();
-        $session = static::$kernel->getContainer()->get('session');
 
         $client->request('GET', '/flash');
-        $this->assertFalse($session->isStarted());
+        if (Kernel::MAJOR_VERSION < 6) {
+            $session = static::$kernel->getContainer()->get('session');
+            $this->assertFalse($session->isStarted());
```

Comment:
is there a way to check that no session is started in symfony 6? the upgrade instructions say the session is only available from the request stack, but the stack is empty at this point here.